dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/Metrics.java

```diff
@@ -14,16 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.dubbo.monitor.support;
+package org.apache.dubbo.metrics;
 
-import org.apache.dubbo.monitor.MetricsService;
-import org.apache.dubbo.rpc.model.BuiltinServiceDetector;
+import org.apache.dubbo.common.Node;
 
-public class MetricsServiceDetector implements BuiltinServiceDetector {
-
-    @Override
-    public Class<?> getService() {
-        return MetricsService.class;
-    }
+/**
+ * Monitor. (SPI, Prototype, ThreadSafe)
+ *
+ * @see org.apache.dubbo.metrics.MetricsFactory#getMetrics(org.apache.dubbo.common.URL)
+ */
+public interface Metrics extends Node, MetricsService {
```

Comment:
`MetricsProvider` should not implement `MetricsService`. I think `MetricsService` is a service that provides metrics query, not implemented by metrics provider.

Reply:
deleted